### DISCUSSION TITLE
Layout that mimics Pop!_OS' tiling mode.

### DISCUSSION THREAD

**Discussion by lominoss-git** [qtile/qtile#3313] [Ideas] [answered]
I'm just looking for a layout that uses the **Fibonacci sequence**.

PS: I tried all the built-in layouts but none of them achieved what I'm looking for.
Thanks.

**Comment by elParaguayo** [comment]
Are you looking to have a layout that spirals (e.g. something like the diagram at the top of this page: https://morioh.com/p/73ea8b225471)?

**Reply by lominoss-git** [reply]
Yes that's exactly what I'm going for!

**Reply by elParaguayo** [reply]
OK - I'm happy to take a look at adding this.

**Reply by lominoss-git** [reply]
You're the man my dude I really appreciate you even considering it. Thank you so much.

**Reply by elParaguayo** [reply]
I'll see if I can do a "proof of concept" and we can then think about what needs to be added in terms of functionality (i.e. growing individual windows may be tricky here)

**Reply by elParaguayo** [reply]
*VERY* rough example:

https://user-images.githubusercontent.com/946265/152992593-beebefd6-10a1-4249-b016-7b3b6fa5c003.mp4

**Reply by lominoss-git** [reply]
I'm sorry but the video file seems to be corrupt.

**Reply by elParaguayo** [reply]
Oh. Plays ok for me on two computers.

EDIT: Interestingly it doesn't play on my phone.

**Reply by lominoss-git** [reply]
<img width="1005" alt="Capture" src="https://user-images.githubusercontent.com/79030093/152994036-3dbd6628-d623-4ae6-a02e-3e688f5669f8.PNG">

**Reply by elParaguayo** [reply]
OK. Let's try again. Trying h264 encoding. Hmm. Also doesn't work on my phone.


https://user-images.githubusercontent.com/946265/152994366-62004afb-c8a5-43e1-aca9-e8f88d6483de.mp4

**Reply by lominoss-git** [reply]
<img width="812" alt="Capture" src="https://user-images.githubusercontent.com/79030093/152994602-75d1296d-685f-4d3f-9dd6-0b6a500038d5.PNG">
A photo will do.

**Reply by elParaguayo** [reply]
![20220208_131912](https://user-images.githubusercontent.com/946265/152995059-814a0d7f-4664-4166-ad51-44af50673d06.png)

**Reply by lominoss-git** [reply]
This is exactly it, well done my dude.

**Reply by lominoss-git** [reply]
Looking forward to the update that adds this layout. Great job my friend.

**Reply by elParaguayo** [reply]
Another screenshot. Easier to see the spiral with added margins.

![20220208_180307](https://user-images.githubusercontent.com/946265/153048452-b6a5c9d8-8b85-4fdb-9807-3b23e4447b3e.png)

**Reply by lominoss-git** [reply]
Oooh man! This looks truly fantastic. And I can see how the margins amplify the effect.

**Comment by elParaguayo** [comment]
The delay here is that I'm trying to work out an algorithm for displaying the windows as I can see 8 potential ways the windows could be laid out:
 - Main pane: Left, Right, Top, Bottom:
 - Spiral direction: clockwise, anti-clockwise

**Reply by lominoss-git** [reply]
That would make it really cool and extremely expandable and customizable,and by the way I really appreciate the amount of time, though and work you are putting into this. God speed my friend.

**Reply by elParaguayo** [reply]
Was being dumb. Think I've got a solution for this.

**Comment by elParaguayo** [comment, marked as answer]
https://github.com/qtile/qtile/pull/3331

**Reply by lominoss-git** [reply]
Marvelous, thank you so much!
